dcpy/connectors/edm/recipes.py

```diff
@@ -92,61 +124,68 @@ def get_latest_version(name):
     return get_config(name)["dataset"]["version"]
 
 
-def fetch_sql(ds: Dataset, local_library_dir):
-    """Retrieve SQL dump file from edm-recipes. Returns fetched file's path."""
+def fetch_dataset(ds: Dataset, local_library_dir=LIBRARY_DEFAULT_PATH) -> Path:
```

Comment:
LIBRARY_DEFAULT_PATH is using relative dcpy paths, is that something we want to clear out now that we're installing dcpy as a package? I do like it being in the root of the directory, maybe that's something we should be setting as an env var somehow?

Reply:
I think setting root path as an env var is a great idea. And since this is the only remaining usage for the *_ROOT_PATH stuff in `dcpy.__init__` it might be time to ✂️. For now I'll just remove it, and just use the working directory when `local_library_dir` isn't specified.